sass/tji/_variables.scss

```diff
@@ -1,10 +1,50 @@
 /* colors */
 $primaryBlue: #0B5D93;
-	$secondaryBlue: #094D79;
+  $secondaryBlue: #094D79;
 $primaryRed: #CE2727;
-	$secondaryRed: #9B1D1D;
+  $secondaryRed: #9B1D1D;
 $primaryYellow: #FFFD00;
 
 /* fonts */
 $headingFont: 'museo', Arial, sans-serif;
-$bodyFont: 'Raleway', sans-serif; //400, 600 weight
\ No newline at end of file
+$bodyFont: 'Raleway', sans-serif; //400, 600 weight
+
+
+/* classes for extension */
+
+@mixin bootstrap-button() {
+  border-radius: .25rem;
+  cursor: pointer;
+  display: inline-block;
+}
+
+
+%bootstrap-button-light {
+  @include bootstrap-button();
+  color: #212529;
+  background-color: #f8f9fa;
+  border-color: #f8f9fa;
+  &:hover {
+      background-color: #e2e6ea;
+      border-color: #dae0e5;
+  }
+  &:active {
+    background-color: #dae0e5;
+    border-color: #d3d9df
+  }
+}
+
+%bootstrap-button-secondary {
```

Comment:
Glad you're reusing bootstrap mixins but not sure how I feel about look n feel of the button. I'm pretty UI up-tight, but punting on 'design' issues for now. I feel like we can review this in one big pass later and adjust styles then.